packages/contracts/foundry.toml

```diff
@@ -0,0 +1,22 @@
+[profile.default]
+src = 'contracts'
+out = 'out'
+libs = ['node_modules', 'lib']
+test = 'test/foundry'
+cache_path  = 'forge-cache'
+solc_version = '0.8.20'
+evm_version = 'paris'
+fs_permissions = [
+    { access = "read", path = "./test/"},
+    { access = "read-write", path = "./test/"}
+]
```

Comment:
This option is not necessary, as there are no Forge tests or Forge script to read or write files.